config/routes.rb

```diff
@@ -1,11 +1,15 @@
 Rails.application.routes.draw do
-  mount RailsAdmin::Engine => '/admin', as: 'rails_admin'
-  namespace :api, :defaults => {:format => 'json'}  do
+  mount RailsAdmin::Engine => "/admin", as: "rails_admin"
+  namespace :api, :defaults => {:format => "json"}  do
```

Comment:
Unnecessary spacing detected.<br>Use the new Ruby 1.9 hash syntax.<br>Space inside { missing.<br>Space inside } missing.